basiclibrary/src/main/java/basiclibrary/Library.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package basiclibrary;
+import java.util.Random;
+
+public class Library {
+    public boolean someLibraryMethod() {
+        return true;
+    }
+
+
+    public static int[] roll(int num){
+        int[] rollArray = new int[num];
+        for(int i = 0; i < num; i++) {
+            rollArray[i] = (int) (1 + Math.random() * (6 - 1));
```

Comment:
Try to avoid using magic numbers (1 and 6). Instead, you can instantiate two int variable: min and max. This will make the code more clear and make later updates easier, like if we decide to use a 12 sided dice.